wizard/WizardManageWalletUI.qml

```diff
@@ -297,18 +301,31 @@ ColumnLayout {
             placeholderFontBold: true
             placeholderFontFamily: "Arial"
             placeholderColor: Style.legacy_placeholderFontColor
-            placeholderText: qsTr("Restore height (optional)") + translationManager.emptyString
+            placeholderText: qsTr("Wallet creation date in YYYY-MM-DD or Restore height (optional)") + translationManager.emptyString
```

Comment:
Worth to mention that a user might specify an approximate date

Reply:
Any wording suggestions? Sounds like a good idea to me.